hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -63,6 +64,11 @@ def _default_employee(self):
             ('company_id', '=', company.id),
         ], limit=1)
 
+    sheet_name = fields.Char(
+        string='Sheet Name',
+        compute='_compute_sheet_name',
+        context_dependent=True,
```

Comment:
?? This is not needed. And does it exist?

Reply:
https://github.com/odoo/odoo/blob/44515bc7beae3b4c46b5bc2959c320695b51439d/odoo/fields.py#L320 it does

Regarding the need of that - not sure about that, locale is in context?

Reply:
`'context_dependent': False, # whether the field's value depends on context.` according to the source code - it should be true @pedrobaeza what's your though on that? Since right now we're only relying on value not-being-stored, yet if I'll change locale in context - the cache won't be invalidated?